.github/workflows/integration_tests.yml

```
@@ -0,0 +1,73 @@
+name: 'Integration Tests'
+
+#This github action runs all of Cromwell's integration tests.
+
+#This is what shows up in the github workflows page as the title. 
+run-name: ${{ github.actor }} Integration Testing.
+
+#What will trigger the workflow to run. 
+on:
+  workflow_dispatch: #Manual trigger from GitHub UI
+  push: #git push
+
+permissions: 
+  contents: read
+
+jobs:
+  integration-tests:
+    strategy:
+      fail-fast: false #disabling fail-fast means that even if one test fails, the others will still try to complete.
+      #Each entry below is a single integration test that lives in /src/ci/bin/. 
+      #Each will be launched on its own runner so they can occur in parallel.
+      #Friendly names are displayed on the Github UI and aren't used anywhere else.
+      matrix:
+        include:
+          - build_type: centaurPapiV2beta
+            build_mysql: 5.7
+            friendly_name: Centaur Papi V2 Beta with MySQL 5.7
+          - build_type: dbms
+            friendly_name: DBMS
+          - build_type: centaurTes
+            build_mysql: 5.7
+            friendly_name: Centaur Tes with MySQL 5.7
+          - build_type: centaurLocal
+            build_mysql: 5.7
+            friendly_name: Centaur Local with MySQL 5.7
+          - build_type: checkPublish
+            friendly_name: Check Publish
+          - build_type: centaurAws
+            build_mysql: 5.7
+            friendly_name: Centaur AWS with MySQL 5.7
+          - build_type: centaurDummy
+            build_mysql: 5.7
+            friendly_name: Centaur Dummy with MySQL 5.7
+          - build_type: centaurHoricromtalPapiV2beta
+            build_mysql: 5.7
+            friendly_name: Centaur Horicromtal PapiV2 Beta with MySQL 5.7
+          - build_type: horicromtalDeadlock
+            friendly_name: Horicromtal Deadlock
+          - build_type: singleWorkflowRunner
+            friendly_name: Single Workflow Runner
+    name: ${{ matrix.friendly_name }}
+    env:
+      BUILD_NAME: ${{ matrix.build_type }}
+      BUILD_TYPE: ${{ matrix.build_type }} #intentionally duplicated variable
+      BUILD_MYSQL: ${{ matrix.build_mysql }}
+      BUILD_POSTGRESQL: ${{ matrix.build_postgresql }}
+      BUILD_MATRIADB: ${{ matrix.build_mariadb }}
```

typo, may lead to unexpected outcomes

```suggestion
      BUILD_MARIADB: ${{ matrix.build_mariadb }}
```